test/Azure.Functions.Cli.Tests/E2E/CreateFunctionTests.cs

```diff
@@ -12,19 +12,21 @@ public class CreateFunctionTests : BaseE2ETest
     {
         public CreateFunctionTests(ITestOutputHelper output) : base(output) { }
 
-        [Fact]
-        public async Task create_template_function_success_message()
+        [Theory]
+        [InlineData("dotnet", "HttpTrigger")]
+        [InlineData("dotnet", "KafkaTrigger")]
```

Comment:
I don't think this test is meant for every trigger. If it was, it should call `func init` once and then iterate through all the triggers on the same project to save time. For example, the Kafka test as-is took about ~17 seconds and I feel like anything over 2 is long for a unit test

I don't have a super strong preference on how you handle the test for this PR, but if it were me I would skip the unit test for now and investigate a "create function" test for every trigger type later. Depending how much time that added to each test run, I might prioritize trigger types based on usage.

Reply:
That makes sense. I will remove the test for now since it seems to be taking a lot of time, then.